toolkit/imageconfigs/linuxguard-amd64.yaml

```diff
@@ -119,14 +119,12 @@ os:
       - rd.luks=0
       - rd.hostonly=0
       - ipe.enforce=0
-      - systemd.journald.forward_to_console=1
 
   packages:
     remove:
       - dracut-hostonly
-      - grub2-efi-binary
       - kernel
-
+      - grub2-efi-binary
```

Comment:
Curious why the reorder?

Reply:
The only reason is because this is the order in Steamboat